spec/rules_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rules'
+
+  describe Rules do
+
+    include Rules
+
+    let(:dummy_class) { Class.new { extend Rules } }
+    let(:strike) { double(:strike, strike?: 10) }
+    let(:spare) { double(:spare, spare?: 10) }
+    let(:pins_hit) { double(:pins_hit, pins_hit?: 5) }
+    let(:spare) { double(:spare, spare?: 10) }
+    let(:gutter) { double(:gutter, gutter?: 0) }
+    let(:both_throw) { double(:both_throw, both_throws: 7) }
+
+    it 'defaults the number of pins to 10' do
+      expect(pins_hit.pins_hit?).to eq 5
+    end
+
+    it 'strike is set as 10 pins being hit' do
+      expect(strike.strike?).to eq 10
+    end
+
+    it 'spare is set at 10 pins being hit in total across two turns' do
+      expect(spare.spare?).to eq 10
+    end
+
```

Comment:
Good testing for spare edge cases, you could probably encapsulate the spare tests into one.